pkg/router_test.go

```diff
@@ -32,7 +32,7 @@ func TestRouter(t *testing.T) {
 	done := make(chan bool)
 	onTrackFired, onTrackFiredFunc := context.WithCancel(context.Background())
 	pubsfu.OnTrack(func(track *webrtc.Track, _ *webrtc.RTPReceiver) {
-		receiver := NewWebRTCVideoReceiver(ctx, WebRTCVideoReceiverConfig{}, track)
+		receiver := NewWebRTCReceiver(ctx, track).(*WebRTCReceiver)
```

Comment:
Seems having the `Sender` `Receiver` interface doesn't really buy us much. Unless we plan to support other transports but could be awhile before that makes sense. Should we just use the `WebRTC*` types for now?

Reply:
I think they can help for mocking for uni testing, at the moment testing some parts like router becames complicated to test because we need to take care of peer signaling that almost become an e2e test.